app/screens/question/question-screen.tsx

```diff
@@ -50,20 +51,19 @@ const CHECK_ANSWER: ViewStyle = {
 }
 
 export const QuestionScreen = observer(function QuestionScreen() {
-  // Are we refreshing the data
   const [refreshing, setRefreshing] = useState(false)
 
   // Pull in one of our MST stores
   const { questionStore } = useStores()
   const { questions } = questionStore
 
   useEffect(() => {
-    fetchQuestions()
+    setTimeout(fetchQuestions, 1)
```

Comment:
Can you add a comment here with the link to the Ignite issue?

Reply:
Yep 👍